generators/paths/domain_parser/code_snippet_common.md

```diff
@@ -0,0 +1,36 @@
+```python
+import re
+
+def domain_parser(link:str, include_sub_domain:bool = False) -> str:
+    """
+    @param link: the input link
+    @param include_sub_domain: Attribute whether subdomain in front of domain is desired or not
+    @return: rootdomain
+    """
+
+    clean_link = re.sub("www.", "",link)
```

Comment:
why is this a re.sub? a simple string replace should do the same if we don't use regular expressions